### ISSUE TITLE
[BUG] 1.6.2 breaks: fatal error: 'benchmark/export.h' file not found

### ISSUE THREAD

**Issue by yurivict** [google/benchmark#1436]
```
Run Build Command(s):/usr/local/bin/ninja cmTC_14e94 && [1/2] Building CXX object CMakeFiles/cmTC_14e94.dir/thread_safety_attributes.cpp.o
FAILED: CMakeFiles/cmTC_14e94.dir/thread_safety_attributes.cpp.o 
/usr/bin/c++   -O2 -pipe -fno-omit-frame-pointer -fstack-protector-strong -fno-strict-aliasing -fno-omit-frame-pointer  -Wno-thread-safety-analysis -Wno-sign-compare  -std=c++11  -Wall  -Wextra  -Wshadow  -Wfloat-equal  -Wsuggest-override  -pedantic  -pedantic-errors  -Wshorten-64-to-32  -fstrict-aliasing  -Wno-deprecated-declarations  -Wstrict-aliasing  -Wthread-safety -MD -MT CMakeFiles/cmTC_14e94.dir/thread_safety_attributes.cpp.o -MF CMakeFiles/cmTC_14e94.dir/thread_safety_attributes.cpp.o.d -o CMakeFiles/cmTC_14e94.dir/thread_safety_attributes.cpp.o -c /disk-samsung/freebsd-ports/devel/benchmark/work/benchmark-1.6.2/cmake/thread_safety_attributes.cpp
In file included from /disk-samsung/freebsd-ports/devel/benchmark/work/benchmark-1.6.2/cmake/thread_safety_attributes.cpp:2:
In file included from /disk-samsung/freebsd-ports/devel/benchmark/work/benchmark-1.6.2/cmake/../src/mutex.h:7:
/disk-samsung/freebsd-ports/devel/benchmark/work/benchmark-1.6.2/cmake/../src/check.h:8:10: fatal error: 'benchmark/export.h' file not found
#include "benchmark/export.h"
         ^~~~~~~~~~~~~~~~~~~~
1 error generated.
ninja: build stopped: subcommand failed.
```

BUILD_SHARED_LIBS=ON BENCHMARK_ENABLE_TESTING=OFF

clang-13
FreeBSD 13.1

**Comment by dominichamon** [comment]
are you at HEAD? i'm not seeing `-I include/` that i would expect on your command line.

i assume you generated the ninja build from cmake? i have just done that locally (clang-13, debian (ish)) and i can't reproduce your issue.

**Comment by sonullx** [comment]
`include/benchmark/export.h` is missing in tag v1.6.2

**Comment by yurivict** [comment]
If the release is broken - you should remove it and make the next release with a higher number.
Otherwise people would get confused.

**Comment by dominichamon** [comment]
oh wow, you're right. i have no idea how github managed to create a release with a file missing.

**Comment by dominichamon** [comment]
released v1.7.0
